.github/workflows/admin-sample.cd.yml

```
@@ -263,6 +263,17 @@ jobs:
         dotnet new install Bit.Boilerplate.0.0.0.nupkg
         cd ../../../ && dotnet new bit-bp --name AdminPanel --database PostgreSQL --sample Admin --appInsights --sentry --serverUrl ${{ env.SERVER_ADDRESS }} --filesStorage AzureBlobStorage --notification --captcha reCaptcha --signalR --framework net9.0
   
+    - name: Delete AssemblyOriginatorKeyFile.snk
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1001:info:1:15: This \s will be a regular 's' in this context

(shellcheck)